src/Catalyst.Core.Lib/P2P/ReputationSystem/ReputationManager.cs

```diff
@@ -64,15 +64,9 @@ public void MergeReputationStream(IObservable<IPeerReputationChange> reputationC
             MergedEventStream = ReputationEventStream.Merge(reputationChangeStream);
         }
 
-        private void OnCompleted()
-        {
-            _logger.Debug("Message stream ended.");
-        }
+        private void OnCompleted() { _logger.Debug("Message stream ended."); }
 
-        private void OnError(Exception obj)
-        {
-            _logger.Error("Message stream ended.");
-        }
+        private void OnError(Exception obj) { _logger.Error("Message stream ended."); }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unused method parameter 'obj'.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4614986)